app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,173 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light-gray: #f2f2f2;
+$white: $white;
+$beach-blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index-background-header {
+  color: $white;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 5em;
+  font-weight: bold;
+  padding-top: 300px;
+  text-align: center;
+}
+
+#search-bar-container .well {
+  background: rgba(0, 0, 0, 0.1);
+  border-color: transparent;
+  bottom: 0;
+  left: 0;
+  margin-bottom: 0;
+  position: absolute;
+  right: 0;
+}
+
+.columns {
+  padding-top: 150px;
+}
+
+.index-containers {
+  margin-top: 100px;
+  p {
+    color: inherit;
+  }
+  h1 {
+    color: inherit;
+  }
+  * {
+    color: $white;
+  }
+}
+
+
+
+#beach-snapshot {
+  span {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets